lib/compat/wordpress-5.9/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -282,7 +282,7 @@ public static function get_user_data() {
 		}
 
 		$config   = array();
-		$user_cpt = self::get_user_data_from_custom_post_type( wp_get_theme() );
```

Comment:
In line 277, there's a missing change:

- before: `@return WP_Theme_JSON_Gutenberg Entity that holds user data.`
- after: `@return WP_Theme_JSON_Gutenberg Entity that holds styles for user data.`

Reply:
Updated in 63608f6.